mozilla-release/mobile/android/base/java/org/mozilla/gecko/AddonUICache.java

```diff
@@ -289,6 +289,11 @@ private void updateAddonMenuItem(String uuid, final GeckoBundle options) {
      */
     private static void addAddonMenuItemToMenu(final Menu menu, final MenuItemInfo info) {
         final Menu destination;
+        /* Cliqz Start */
```

Comment:
It's better to handle it, inside handleMessage() , case "Menu:Add"
don't added it if it's Ghostry, so we save some memory by prevent adding instead of prevent showing.

Reply:
Ah right. It makes sense to do that.